runtime/src/accounts.rs

```diff
@@ -690,6 +690,15 @@ impl Accounts {
     }
 }
 
+pub fn create_accounts(accounts: &Accounts, pubkeys: &mut Vec<Pubkey>, num: usize) {
```

Comment:
pub(crate) and rename to `create_test_accounts` ?

Reply:
Ah nice!  I didn't realize there was control of `pub`ness.  ACK ACK